tests/capture_tools_output/test_database_path.py

```diff
@@ -53,23 +54,19 @@ def test_db_detection(
         extra_args=[],
         tidy_review=False,
         format_review=False,
+        num_workers=2,
```

Comment:
Correct, I just wanted to ensure multiple processes to be spawned. One process will run clang-tidy and the other will just sleep, so might not be an issue?

Reply:
The test should reflect the performance of the machine running it. And since we're not actually testing `main()`, then I think it is appropriate to set this dynamically, even if only 1 process is used.

Reply:
Ok, then this should be done in the command line parser.

Reply:
Fixed in 0e78226dcd10aa2b9df60e588a081d4cb15d434f.